FirstProject/assets/common.less

```diff
@@ -0,0 +1,18 @@
+.button {
+  border: none;
```

Comment:
![colors](https://user-images.githubusercontent.com/31060543/29561751-62a29042-873f-11e7-8129-843c22c37577.png)

something like that?